### PR TITLE
fix regression from cc9977, reference to data-query-index #89

### PR DESCRIPTION
**Pull request by peller**
(c) Copyright IBM Corp. 2016

### CODE REVIEW

File: urth-viz-query.html [hunk 1]
```diff
@@ -176,7 +176,7 @@
                         </template>
                     </div>
 
-                    <paper-button class="add-agg" on-tap="_addAgg" noink>
+                    <paper-button class="add-agg" on-tap="_addAgg" noink data-query-index$="{{query-index}}">
```
Comment (dalogsdon) [inline urth-viz-query.html]: I think this also needs to be added to the `.agg-remove-btn` element above.

File: urth-viz-query.html [hunk 2]
```diff
@@ -437,7 +429,7 @@
         },
 
         _removeAgg: function(event) {
-            var i = getQueryIndex(event.currentTarget);
+            var i = event.currentTarget.getAttribute("data-query-index");
```
Comment (dalogsdon) [inline urth-viz-query.html]: I am getting an error here when I try to remove an aggregation. See above comment about adding attribute to `.agg-remove-btn`.

File: urth-viz-query.html (hunk 1 above)
Comment (peller) [inline urth-viz-query.html]: good catch

**Comment by dalogsdon** [comment]
Looks good now.

**Pull request merged**